src/components/blipDatepicker/index.js

```diff
@@ -257,8 +255,7 @@ export class BlipDatepicker extends Component {
 
     const timeInputContainer = html`
     <div class="${BlipDatepicker.style.timeInputContainer}">
-      <div class="${BlipDatepicker.style.clockIcon}">${raw(Clock)}</div>
-      <input class="${BlipDatepicker.style.timeInput}" type="time" required>
+      <input class="${BlipDatepicker.style.timeInput}" type="date" required />
```

Comment:
This input expects a time format, does date type gives this format?

Reply:
No. I'll fix it and create a new pull request.

Reply:
okkk

Reply:
You can commit in this same PR